ETH/Overview.md

```diff
@@ -0,0 +1,42 @@
+# P14 ETH - Overview
+### 1 Definition
+
+Bitcoin and Ethereum are two major cryptocurrencies.
+
+| name              | definition                                          | unit    |
+| ----------------- | --------------------------------------------------- | ------- |
+| Bitcoin （BTC）   | decentralized currency （known as blockchain 1.0）  | Satoshi |
+| Ethereum  （ETH） | decentralized contract  （known as blockchain 2.0） | Wei     |
+
+### 2 Optimization of ETH compared to BTC
+
+Ethereum has made improvements to Bitcoin's operational issues in system design:
+
+- Shortened block generation time from 10 minutes to just over ten seconds and designed a consensus mechanism based on the **GHOST protocol**.
+
+- Changed mining puzzle: Bitcoin's is computationally intensive for hash value calculation with specialized mining equipment. Ethereum's is memory hard to resist ASIC chips. Future change will replace proof of work with proof of stake.
+
+In addition, Ethereum supports **smart contracts**.
+
+### 3 What is a decentralized contract? 
+
+- After Bitcoin's success, people wonder what else can be decentralized.The emergence of Ethereum has added the characteristic of decentralized contracts. 
+- Bitcoin is decentralized currency, replacing government functions with technical means like cryptography and consensus mechanisms.
+- Ethereum has decentralized contracts. In real society, contracts are maintained by government through judicial means. Smart contracts on Ethereum replace judicial means with technical means.
+- Some contracts with simple and clear logic can be written as smart contracts, but not all contract content can be implemented in programming languages or quantified.
+
+### 4 What are the benefits of decentralized contracts?
+
+- Decentralized currency like Bitcoin has an advantage in cross-border transfers compared to legal tender as it is faster despite some imperfections.
+
+- In contracts with signatories from all over and no unified jurisdiction, judicial means are difficult for contract enforcement. Online crowdfunding with participants worldwide is an example.Even under the same jurisdiction, judicial contract enforcement is time-consuming and laborious. Winning a lawsuit doesn't guarantee getting money.
```

Comment:
Make it concise is better. Like just a few points, like immutable, "must execute according to the code rules". You should make it look better, in a structured way.